### PR TITLE
Removes the druggy rainbow overlay because it fucked my eyes.

### PR DESCRIPTION
**Pull request by BurgerLUA**
## About The Pull Request

Removes the druggy rainbow overlay from being high.

## Why It's Good For The Game

Hurts my eyes.

## Proof Of Testing

If it compiles, it works.

## Changelog

:cl: BurgerBB
del: Removes the druggy rainbow overlay because it fucked my eyes.
/:cl:

### CODE REVIEW

File: code/datums/status_effects/debuffs/drugginess.dm [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	RegisterSignal(owner, COMSIG_LIVING_DEATH, PROC_REF(remove_drugginess))
 
 	owner.add_mood_event(id, /datum/mood_event/high)
-	owner.overlay_fullscreen(id, /atom/movable/screen/fullscreen/high)
+	// owner.overlay_fullscreen(id, /atom/movable/screen/fullscreen/high) BUBBERSTATION CHANGE: REMOVES HIGH OVERLAY.
```
Comment (lessthnthree) [inline code/datums/status_effects/debuffs/drugginess.dm:15]: If you hate it that much, code it to be a toggleable game preference.

**Review by lessthnthree — CHANGES_REQUESTED** [review]

File: code/datums/status_effects/debuffs/drugginess.dm (hunk 1 above)
Comment (BurgerLUA) [reply code/datums/status_effects/debuffs/drugginess.dm:15]: way too much effort my dude for something that is annoying

**Comment by BurgerLUA** [comment]
????

**Comment by Swiftfeather** [comment]
> ????

You said it was too much effort to make the requested change. Yeah, it's annoying, because your character has **done drugs**.

**Comment by BurgerLUA** [comment]
> > ????
> 
> You said it was too much effort to make the requested change. Yeah, it's annoying, **because your character has done drugs**.

It happens whenever you breathe in trace amounts of n2o. I did not willingly take Space Drugs.

**Comment by Swiftfeather** [comment]
> > > ????
> > 
> > 
> > You said it was too much effort to make the requested change. Yeah, it's annoying, **because your character has done drugs**.
> 
> It happens whenever you breathe in trace amounts of n2o. I did not willingly take Space Drugs.

There's two parts of cause and effect, look at the cause. https://github.com/tgstation/tgstation/pull/81223

**Comment by BurgerLUA** [comment]
> > > > ????
> > > 
> > > 
> > > You said it was too much effort to make the requested change. Yeah, it's annoying, **because your character has done drugs**.
> > 
> > 
> > It happens whenever you breathe in trace amounts of n2o. I did not willingly take Space Drugs.
> 
> There's two parts of cause and effect, look at the cause. [tgstation/tgstation#81223](https://github.com/tgstation/tgstation/pull/81223)

I'm not sure what this means. Is this intentional behavior? The amount you need to breathe in is still very tiny to have an effect.

**Pull request closed without merging**